app/src/main/AndroidManifest.xml

```diff
@@ -23,8 +24,10 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_dessert_clicker_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-        <activity android:name="com.example.android.dessertclicker.MainActivity">
+        android:theme="@style/AppTheme"
+        tools:ignore="AllowBackup">
```

Comment:
Is this attribute necessary? I'm not sure what it does, so I was wondering if we could omit it?